manager/gui/src/lib/components/SgtDataTable/SgtDataTable.vue

```diff
@@ -389,12 +391,13 @@ export default class SgtDataTable extends Vue {
    * On chip cancel button click. event emitted from chips component.
    */
   updateFilterByChip(chip: SgtFilterObject) {
-    const ind = this.tableDataConfig.filterList.findIndex(
+    const formField = this.searchConfig.advanceForm?.find(
       (ele) => ele.name == chip.name
     );
-    if (ind >= 0) {
-      this.tableDataConfig.filterList.splice(ind, 1);
-      this.updateRecord();
+    if (formField) {
+      formField.value = "";
```

Comment:
Make this null or undefined, will handle cases for type number, checkboxes, dropdowns, radios, etc.

Reply:
Assigned 'null' to this variable.